### ISSUE TITLE
Выполнять запрос /finish при завершении обстрела

### ISSUE THREAD

**Issue by orangy** [sat2707/hlcupdocs#16]
Это было бы полезно для вывода в лог агрегированной информации по производительности. После ответа на этот запрос завершать процесс.

**Comment by ei-grad** [comment]
А на SIGTERM если завязаться, не получится вывести что нужно при завершении?

**Comment by AterCattus** [comment]
Пробовал после INT/TERM писать в stdout, в выдаче не вижу. Так что не понятно (хотя не тестил особо).
А получалось у кого?

**Comment by densmnko** [comment]
Так, вроде, заранее известно кол-во запросов для каждой из фаз и есть признак типа прогона в options.txt. Я просто считаю запросы и по этим счетчикам определяю фазы...
